document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImpl.java

```diff
@@ -26,25 +27,25 @@
     private static final Logger LOG = LoggerFactory.getLogger(MODULE_NAME_SPACE);
 
     @Override
-    public DocumentInfo getDocumentInfo() {
+    public DocumentInfoResponse getDocumentInfo(DocumentInfoRequest documentInfoRequest) {
         LOG.info("Started getting document");
 
-        String resource = "";
-        String resourceId = "";
+        String resourceId = documentInfoRequest.getResourceId();
+        String resourceUri = documentInfoRequest.getResourceUri();
 
-        Transaction transaction = transactionService.getTransaction(resource);
+        Transaction transaction = transactionService.getTransaction(resourceId);
```

Comment:
This should be the resourceURI. The resource URI is the self link in the transaction.